view/index.html

```diff
@@ -1,14 +1,129 @@
 <!DOCTYPE html>
-<html lang="en-GB">
+<html class="core" lang="en-GB">
 <head>
 	<meta charset="utf-8" />
+	<meta http-equiv="X-UA-Compatible" content="IE=edge" />
 	<title>Origami Image Service Tests</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+	<script>
+		var cutsTheMustard = ('querySelector' in document && 'localStorage' in window && 'addEventListener' in window);
+		if (cutsTheMustard) {
+			document.documentElement.className = document.documentElement.className.replace(/\bcore\b/g, 'enhanced');
+		}
+	</script>
+
+	<style>
+		.core .o--if-js,
+		.enhanced .o--if-no-js { display: none !important; }
+		html {
+			font-family: Georgia, serif;
+			overflow-x: hidden;
+		}
+		body {
+			margin: 0;
+		}
+		.origami-logo {
+			display: inline-block;
+			width: 40px;
+			height: 40px;
+			background-repeat: no-repeat;
+			background-position: 50% 50%;
+			background-size: contain;
+			background-image: url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0naHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmcnIHZpZXdCb3g9JzAgMCA2My4zIDYzLjMnPjxnPjxnPjxwb2x5Z29uIGZpbGw9JyNEN0QxQzYnIHBvaW50cz0nMzEuNyw1MS4zIDE5LjksNjMuMiAwLjEsNDMuNCAwLjEsMTkuNycvPjxwb2x5bGluZSBmaWxsPScjQjdCNUFCJyBwb2ludHM9JzQzLjYsMCAxOS45LDAgMC4xLDE5LjcgMTIsMzEuNiA0My42LDAnLz48cG9seWxpbmUgZmlsbD0nIzg3ODY4MCcgcG9pbnRzPSc2My4zLDQzLjQgNjMuMywxOS43IDQzLjYsMCAzMS43LDExLjggNjMuMyw0My40Jy8+PHBvbHlnb24gZmlsbD0nIzY0NjQ2NCcgcG9pbnRzPScxOS45LDYzLjIgNDMuNiw2My4yIDYzLjMsNDMuNCA1MS41LDMxLjYnLz48cGF0aCBmaWxsPScjRDdEMUM2JyBkPSdtMC4wOTgwNjggMTkuNycvPjwvZz48L2c+PC9zdmc+");
+			vertical-align: top;
+			margin-right: 10px;
+		}
+	</style>
+	<link rel="stylesheet" href="https://origami-build.ft.com/v2/bundles/css?modules=o-fonts@^2,o-techdocs@^5,o-ft-icons@^4,o-grid@^4,o-table@^4" />
```

Comment:
o-ft-icons is v3 I believe. I released o-icons v4, though that will probably break stuff if you try to use it